src/EloquentHashids.php

```diff
@@ -51,4 +51,12 @@ public static function getHashidEncodingValue(Model $model)
   {
     return $model->getKey();
   }
+  
+  /**
+   * @return string
+   */
+  public function getRouteKeyName()
+  {
+      return 'uid';
```

Comment:
What I'm saying is that, under normal circumstances, the route name should be the same as the column name. As such, the `getRouteKeyName` function should return the name of the column (through `getHashidColumn`), which could then be overriden if required.